src/OrchardCore.Modules/Orchard.Autoroute/Handlers/AutoroutePartHandler.cs

```diff
@@ -119,5 +124,29 @@ private string GetPattern(AutoroutePart part)
 
             return pattern;
         }
+
+        private string GenerateUniquePath(string path, AutoroutePart context)
```

Comment:
I feel obliged to insist that we should try to generate the content for the user. That usually adds more issues, from experience. I have a bad feeling here, and the benefits don't convince me.